### ISSUE TITLE
Vbox machines have a french keyboard layout

### ISSUE THREAD

**Issue by denisri** [brainvisa/casa-distro#252]
Thus non-french users are a bit lost, and installing a different layout on Lubuntu appears to be a bit complex:
- right-click on the bar on the bottom of the VM screen, and select "Add/remove panel items"
- select the "Panel applets" tab, then "add"
- select "Keyboard layout handler"
- then you can close the panel items edition dialog. You should see a new icon (with a french flag) at the bottom right of the VM screen.
- right-click on this flag, and select "keyboard layout handler settings"
- un-check "Keep system layout" on the right. Then the left panel will be enabled
- click "add" on the left
- select "English (US)" and close the dialogs
- then a left click on the flag toggles between french and US layouts.

Could we pre-install the US layout (at least) so that users just have to click on the icon on the bottom right of the screen.

**Comment by sapetnioc** [comment]
Yes we can !
I used french by default because that was the only way to avoid mistakes during installation (especially for passwords).
But is US enough ? Each language has its own layout.

**Comment by sapetnioc** [comment]
Another possibility would be to create a shortcut on Desktop (if possible) as well as a README file.

**Comment by denisri** [comment]
> Another possibility would be to create a shortcut on Desktop (if possible) as well as a README file.

Yes the procedure I explained in the ticket allows to install an icon to switch keyboard layouts (desktop applet). We can maybe pre-install a few of them (but which ones?), but anyway it will allow to add new ones more easily and users will see where to do it.
